components/ibm-components/ffdl/serve/README.md

```diff
@@ -0,0 +1,105 @@
+# Seldon Core - Serve PyTorch Model
+
+## Intended Use
+Serve PyTorch Models remotely as web service using Seldon Core
+  
+## Run-Time Parameters:
+Name | Description
+:--- | :----------
+model_id | Required. Model training_id from Fabric for Deep Learning
+deployment_name | Required. Deployment name for the seldon service
+model_class_name | PyTorch model class name', default: 'ModelClass'
+model_class_file | File that contains the PyTorch model class', default: 'model_class.py'
+serving_image | Model serving images', default: 'aipipeline/seldon-pytorch:0.1
+
+## Output:
+Name | Description
+:--- | :----------
+output | Model Serving status
+
+## Sample
+
+Note: the sample code below works in both IPython notebook or python code directly.
+
+### Set sample parameters
+```python
+# Parameters
+model_id = 'Model training_id'
+deployment_name = 'Deployment name for the seldon service'
+model_class_name = 'PyTorch model class name'
+model_class_file = 'File that contains the PyTorch model class'
+serving_image =  'aipipeline/seldon-pytorch:0.1'
+```
+
+```python
+# Additional Parameters
+EXPERIMENT_NAME = 'Seldon Core - Serve PyTorch Model'
+COMPONENT_SPEC_URI = 'https://raw.githubusercontent.com/kubeflow/pipelines/master/components/ibm-components/ffdl/serve/component.yaml'
```

Comment:
Nit: It's probably better to inline this variable into the load_component_from_url call.
Otherwise, with multiple components, it's harder to copy-paste the load_component lines since the variables have the same name.